en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
 
 > **Warning** This topic is under construction. It is not complete and has not been fully reviewed.
 
-During setup you will be asked to assign a Mode Switch on your Remote Control. With it you will be able to switch flight modes during flight and ask the aircraft to act in a different way depending on your needs.
+Flight modes are used to provide different types of autopilot control, enabling manual, assisted and fully autonomous flight.
 
-Depending on the amount of assistance the Flight controller commits to you during flight you will have three main flight modes categories that you can use in PX4.
+Pilots can transition between flight modes using switches on the remote control or with a ground control station (see [Flight Mode Configuration](../config/flight_mode.md)).
 
-   **• Manual Modes**
+This topic provide a very high level view of each of the flight modes, and the (mostly minor) differences in how they behave in copter, fixed-wing and VTOL frames. For more detailed information see [Flight Modes](../flight_modes/README.md).
 
-   **•	Assisted Modes**
+> **Tip** ![GPS](../../images/flight_modes/GPS_s.png) - This symbol is used to denote modes that *require* position information! These modes cannot be enabled without a position lock from GPS, VIO, or some other positioning system.
```

Comment:
@tubeme PX4 now supports VIO and other positioning stuff. So we can't just say "GPS". I still love the icon - unless we can find a better one for positioning (?). Perhaps a target?

Reply:
I will try to make another icon to place besides GPS. GPS is probably the most iconic for positioning device. So let leave it like this just we can add another icon besides for the other positioning devices.

Reply:
Sounds good to me. Even if we had the icon without "GPS" text it might be that little bit more generic.

Reply:
Or the same icon, but "GPS" is replaced by "Position".

It might also be useful to have a similar icon for "requires RC control" - ie an RC controller or gamepad.

Reply:
@tubeme .... some options for icons that might work even better than GPS. I've been thinking it might be nicer to have these icons a fair bit smaller and then have them next to each other as a status list - so you might have GPS, remote control, ... in the bar as a summary.

* ![image](https://user-images.githubusercontent.com/5368500/37748982-ea102b8e-2dd9-11e8-9c5d-d8c9f112a919.png) - [from here](https://www.flaticon.com/free-icon/gps-fixed-indicator_60834)
* ![image](https://user-images.githubusercontent.com/5368500/37749041-2a9dbf0e-2dda-11e8-8592-f1f3d9faabd2.png)  - [from here](https://www.iconfinder.com/icons/126573/gps_location_map_pin_icon#size=32)


* ![image](https://user-images.githubusercontent.com/5368500/37749076-62d29516-2dda-11e8-8757-eb5b5c8cd889.png) - from [here](https://www.google.com.au/search?q=free+small+icon+gps&dcr=0&tbm=isch&tbo=u&source=univ&sa=X&ved=0ahUKEwiTve2Y-P7ZAhVBjpQKHW4ID7kQ7AkINg&biw=1229&bih=587#imgrc=rlid6HG39D63OM:)

And maybe this one for rc:
![image](https://user-images.githubusercontent.com/5368500/37749240-2fff146a-2ddb-11e8-8373-5603ea15b37e.png)
 (from [here](http://www.adelaideadvertising.com/free-drone-icon-design-to-download/)

Reply:
We could also possibly have icons for:
- Not RC controlled
- Auto, Assisted, Manual